### DISCUSSION TITLE
콘솔 table

### DISCUSSION THREAD

**Discussion by HidenLee** [100-hours-a-week/discussions#73] [General]
https://www.codewithshripal.com/tips/javascript/use-console-table-for-better-visualization-of-array-and-objects

디버깅할때 항상 console.log로만 확인했는데 좋은 기능 알게되어 공유합니다.
객체가 한눈에 보여서 편하네요!!

![image](https://github.com/user-attachments/assets/2f939ca5-8f03-4614-9bb6-1ec89f5f4314)

**Comment by ej31** [comment]
세상에 이런 기능이 있었네.. 🤡
